services/compute/compute.go

```diff
@@ -753,8 +755,6 @@ type GalleryImageProperties struct {
 	OsType OperatingSystemTypes `json:"osType,omitempty"`
 	// OsState - This property allows the user to specify whether the virtual machines created under this image are 'Generalized' or 'Specialized'. Possible values include: 'Generalized', 'Specialized'
 	OsState OperatingSystemStateTypes `json:"osState,omitempty"`
-	// HyperVGeneration - The hypervisor generation of the Virtual Machine. Applicable to OS disks only. Possible values include: 'V1', 'V2'
```

Comment:
Any reason to remove this comment?

Reply:
The value will be added in this PR: https://github.com/microsoft/moc-sdk-for-go/pull/96/files. I'll remove this change.

Reply:
still see this change. Can you remove this change?